python/feathub/feature_views/feature_view.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         keep_source_fields: bool = False,
         timestamp_field: Optional[str] = None,
         timestamp_format: str = "epoch",
+        attributes: Optional[Dict[str, str]] = None,
```

Comment:
Do we have a use-case for specifying the per-feature-view attribute in addition to the per-feature attribute?